x-pack/plugins/ml/kibana.json

```diff
@@ -34,7 +34,8 @@
     "kibanaReact",
     "dashboard",
     "savedObjects",
-    "home"
+    "home",
+    "spaces"
```

Comment:
This is marked as a required bundle, but the ML plugin lists the Spaces plugin as an optional dependency.

Reply:
This is fixed with https://github.com/elastic/kibana/pull/83320/commits/21a176cb19fbf9a71d31ee245964808dc946d331
It now checks whether the `spaces` plugin exists.